buildspec.yml

```diff
@@ -0,0 +1,20 @@
+version: 0.2
+env:
+  variables:
+    # Agree to send anonymized reporting data to lambdagems
+    JETS_AGREE: 'yes'
+phases:
+  install:
+    runtime-versions:
+      ruby: 2.6
+    commands:
+      # Jets doesn't yet support ruby 2.7
+      # Let's temporarily install 2.5 until we can upgrade
+      #   https://github.com/tongueroo/jets/issues/444
+      - rbenv install 2.5.8
+      - gem install bundler
+  build:
+    commands:
+      - echo "Starting ${BUILD_PHASE} deploy using $(ruby --version)"
+      - bundle install
```

Comment:
Question: Is there any reason for not having migrations command here.
`bundle exec jets db:migrate`

Reply:
The pantry-finder database is read only. We don't want to execute migrations from the application. Currently, this api exposed a datasource that is modified and loaded externally.

Reply:
got it.

Reply:
@maheshkurma thanks for thinking it through to that extent. For reference, here is our general PT-FT architecture:

![image](https://user-images.githubusercontent.com/11682225/89538460-94e9c200-d7c8-11ea-9c91-33edf6c55f26.png)